doc/lua_api.txt

```diff
@@ -7538,11 +7548,15 @@ Player properties need to be saved manually.
         collide_with_objects = true,
         -- Collide with other objects if physical = true
 
-        collisionbox = {-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
-        selectionbox = {-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
-        -- Selection box uses collision box dimensions when not set.
-        -- For both boxes: {xmin, ymin, zmin, xmax, ymax, zmax} in nodes from
-        -- object position.
+        collisionbox = { -0.5, 0.0, -0.5, 0.5, 0.5, 0.5 },  -- default
+        selectionbox = { -0.5, 0.0, -0.5, 0.5, 0.5, 0.5, rotate = false },
```

Comment:
Whoops, didn't spot this before.
Is the change of `ymin` to 0 intentional?

Reply:
Definitely not. IIRC sfan5 fixed this in a doc fixup PR; I must've merged the changes improperly.